app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/points/GeonamesHandlerTest.java

```diff
@@ -65,9 +66,12 @@
     public static void setup() throws InterruptedException {
         Context context = ApplicationProvider.getApplicationContext();
         Assert.assertNotNull(context);
-        UserPoint userPoint = new UserPoint(MOCK_LOCATION_LAT_LAUSANNE,
-                                            MOCK_LOCATION_LON_LAUSANNE,
-                                            MOCK_LOCATION_ALT_LAUSANNE);
+        UserPoint userPoint = UserPoint.getInstance(context);
+        userPoint.setLocation(MOCK_LOCATION_LAT_LAUSANNE,
+                MOCK_LOCATION_LON_LAUSANNE,
+                MOCK_LOCATION_LON_LAUSANNE,
+                0);
```

Comment:
Error in function call, third parameter should be altitude
```suggestion
        userPoint.setLocation(MOCK_LOCATION_LAT_LAUSANNE,
                MOCK_LOCATION_LON_LAUSANNE,
                MOCK_LOCATION_ALT_LAUSANNE,
                0);
```